docs/dev/dev_tf_plugin.md

```diff
@@ -0,0 +1,50 @@
+## The TFJob plugin framework
+
+If you'd like to customize or enhance the TFJob with your own chart or code.
```

Comment:
If you'd like to customize or enhance the TFJob with your own chart or code, you can consider to  extend it with your own TF runtime.